Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -2239,6 +2239,20 @@ def endpoint_scan_command(client, args):
     alias = args.get('alias')
     isolate = args.get('isolate')
     hostname = argToList(args.get('hostname'))
+    all_ = argToBoolean(args.get('all', 'false'))
+
+    # to prevent the case where an empty filtered command will trigger by default a scan on all the endpoints.
+    err_msg = 'To scan all the endpoints run this command with the \'all\' argument as True ' \
```

Comment:
The error message should mention that the command requires filters to be passed or pass all=true which will trigger scan on all the endpoints but may cause performance impact